benchmarks/dynamo/runner.py

```diff
@@ -682,21 +748,40 @@ def exec_summary_text(self, caption, fn, metric):
-            "these experiments on A100 GPUs. Each experiment runs one iteration of forward "
-            "and backward pass. For accuracy, we check the numerical correctness of forward "
-            "pass outputs and gradients by comparing with native pytorch. We measure speedup "
-            "by normalizing against the performance of native pytorch. We report mean "
-            "compilation latency numbers and peak memory footprint reduction ratio. \n\n"
-            "Caveats\n"
-            "1) Batch size has been reduced to workaround OOM errors. Work is in progress to "
-            "reduce peak memory footprint.\n"
-            "2) Experiments do not cover dynamic shapes.\n"
-            "3) Experimental setup does not have optimizer.\n\n"
-        )
-
+        if "cuda" in self.devices:
+            description = (
+                "We evaluate different backends "
+                "across three benchmark suites - torchbench, huggingface and timm. We run "
+                "these experiments on A100 GPUs. Each experiment runs one iteration of forward "
+                "and backward pass. For accuracy, we check the numerical correctness of forward "
+                "pass outputs and gradients by comparing with native pytorch. We measure speedup "
+                "by normalizing against the performance of native pytorch. We report mean "
+                "compilation latency numbers and peak memory footprint reduction ratio. \n\n"
+                "Caveats\n"
+                "1) Batch size has been reduced to workaround OOM errors. Work is in progress to "
+                "reduce peak memory footprint.\n"
+                "2) Experiments do not cover dynamic shapes.\n"
+                "3) Experimental setup does not have optimizer.\n\n"
+            )
+        else:
+            get_machine_cmd = "lscpu| grep 'Model name' | awk -F':' '{print $2}'"
+            machine = subprocess.getstatusoutput(get_machine_cmd)[1].strip()
+            description = (
+                "We evaluate different backends "
+                "across three benchmark suites - torchbench, huggingface and timm. We run "
+                "these experiments on "
+                + machine
+                + ". Each experiment runs one iteration of forward "
+                "pass. For accuracy, we check the numerical correctness of forward "
+                "pass outputs by comparing with native pytorch. We measure speedup "
+                "by normalizing against the performance of native pytorch. We report mean "
+                "compilation latency numbers and peak memory footprint reduction ratio. \n\n"
+                "Caveats\n"
+                "1) Batch size has been reduced to workaround OOM errors. Work is in progress to "
+                "reduce peak memory footprint.\n"
+                "2) Experiments do not cover dynamic shapes.\n"
+                "3) Experimental setup does not have optimizer.\n\n"
+            )
```

Comment:
These strings seem to be general enough to cover A100 as well?

Reply:
Yes. I refined the description in [6d67af1](https://github.com/pytorch/pytorch/pull/88477/commits/6d67af1197e5e0327f69485a08e6d26ff7ae4fe3)